initalize/initial.go

```diff
@@ -0,0 +1,267 @@
+package initalize
+
+import (
+	"database/sql"
+	"fmt"
+	"sync"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/qustavo/dotsql"
+)
+
+var lockDOT = &sync.Mutex{}
+var lockDB = &sync.Mutex{}
+var initalizedDOT *dotsql.DotSql
+var initalizedDB *sql.DB
+
+const testDBName string = "testdb"
+const prodDBName string = "proddb"
+
+func csvMatches(wg *sync.WaitGroup, db *sql.DB, dot *dotsql.DotSql) {
+	println("Started CSV MATCHES")
+	defer wg.Done()
+	dot.Exec(db, "CSV-DROP-WorldCupMatchesCSV")
+	dot.Exec(db, "CSV-Table-WorldCupMatches")
+	dot.Exec(db, "CSV-Table-WorldCupMatches-Insert")
+	println("Done CSV MATCHES")
+}
+
+func csvPlayers(wg *sync.WaitGroup, db *sql.DB, dot *dotsql.DotSql) {
+	println("Started CSV PLAYERS")
+	defer wg.Done()
+	dot.Exec(db, "CSV-DROP-WorldCupPlayersCSV")
+	dot.Exec(db, "CSV-Table-WorldCupPlayersCSV")
+	dot.Exec(db, "CSV-Table-WorldCupPlayers-Insert")
+	println("Done CSV PLAYERS")
+}
+
+func csvWorldCups(wg *sync.WaitGroup, db *sql.DB, dot *dotsql.DotSql) {
+	println("Started CSV WC")
+	defer wg.Done()
+	dot.Exec(db, "CSV-DROP-WorldCupsCSV")
+	dot.Exec(db, "CSV-Table-WorldCupsCSV")
+	dot.Exec(db, "CSV-Table-WorldCupsCSV-Insert")
+	println("Done CSV WC")
+}
+
+func GetDB() *sql.DB {
+	if initalizedDB == nil {
+		lockDB.Lock()
+		defer lockDB.Unlock()
+		if initalizedDB == nil {
+			fmt.Println("Creating DB now.")
+			db, err := sql.Open("mysql", "root:password@tcp(127.0.0.1:3306)/"+testDBName)
+			if err != nil {
+				panic(err.Error())
+			}
+			initalizedDB = db
+		} else {
+			fmt.Println("DB made already")
+		}
+
+	}
+	return initalizedDB
+}
+
+func GetInstance() *dotsql.DotSql {
```

Comment:
Should run these in parallel with GO coroutines. Please send a design doc for further discussion and review.